src/Http/ApiClient.php

```diff
@@ -13,13 +13,27 @@
 use GuzzleHttp\Client;
 use GuzzleHttp\HandlerStack;
 use GuzzleHttp\Psr7\Uri;
+use phpDocumentor\Reflection\Types\Self_;
 use Psr\Http\Message\ResponseInterface;
 
 /**
  */
 class ApiClient
 {
     /**
+     * These are the endpoints where there is no need to have an active session for the request
+     * to succeed.
+     */
+    const SESSION_SERVER_URL = "session-server";
```

Comment:
Please use single quotes for the constants!

Reply:
Also please post the `URIS_NOT_REQUIRING_ACTIVE_SESSION ` right after the doc header, and the constants right after it :)

Reply:
Java struggles 😁